packages/components/src/isolated-event-container/test/index.js

```diff
@@ -1,29 +1,71 @@
 /**
  * External dependencies
  */
-import { shallow } from 'enzyme';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 /**
  * Internal dependencies
  */
 import IsolatedEventContainer from '../';
 
 describe( 'IsolatedEventContainer', () => {
-	it( 'should pass props to container', () => {
-		const isolated = shallow(
-			<IsolatedEventContainer className="test" onClick="click" />
+	it( 'should pass props to container', async () => {
+		const user = userEvent.setup( {
+			advanceTimers: jest.advanceTimersByTime,
+		} );
+		const clickHandler = jest.fn();
+		render(
+			<IsolatedEventContainer
+				data-testid="container"
+				className="test"
+				onClick={ clickHandler }
+			/>
 		);
 
-		expect( isolated.hasClass( 'test' ) ).toBe( true );
-		expect( isolated.prop( 'onClick' ) ).toBe( 'click' );
+		const container = screen.getByTestId( 'container' );
+		expect( container ).toHaveClass( 'test' );
+
+		await user.click( container );
+
+		expect( clickHandler ).toHaveBeenCalledTimes( 1 );
 		expect( console ).toHaveWarned();
 	} );
 
-	it( 'should stop mousedown event propagation', () => {
-		const isolated = shallow( <IsolatedEventContainer /> );
-		const event = { stopPropagation: jest.fn() };
+	it( 'should render children', async () => {
+		render(
+			<IsolatedEventContainer data-testid="container">
+				<p data-testid="child" />
+			</IsolatedEventContainer>
+		);
+
+		expect(
+			within( screen.getByTestId( 'container' ) ).getByTestId( 'child' )
```

Comment:
Nit: let's try to avoid using `data-testid` when possible. In this case, we could add some text to the `<p />` and use the `getByText` selector instead

Reply:
Good point 👍 Updated to no longer use `data-testid` in 37dab4c64db1e88259e59b7abe46c20b28dfc3ba